x/twap/store.go

```diff
@@ -19,6 +19,8 @@ import (
 // not too small where it would take all the way to the next epoch.
 var NumRecordsToPrunePerBlock uint16 = 200
 
+var PruneLimitPerBlock uint16 = 200
+
```

Comment:
PruneLimitPerBlock is for the temporary requirement of spreading the deletion of the no longer used kvstore over multiple blocks instead of doing it all at once at the upgrade height. NumRecordsToPrunePerBlock is used every epoch for the records that are not deprecated

Reply:
Added clarity to what these two values represent, thanks for pointing out https://github.com/osmosis-labs/osmosis/pull/7779/commits/8c1d14fb5fb1e370ddcbdf744bdb8b5bd4cf6deb